src/Pages/Authentication/Login/index.js

```diff
@@ -0,0 +1,48 @@
+import { Link } from 'react-router-dom';
+
+import './login.css';
+import '../../../styles/base.css';
+
+import { FormInput } from '../../../components/Input';
+import { FormButton } from '../../../components/Button';
```

Comment:
Suggestion: you can make main `index.js` file and import component to there, then you don't have to import components from different files like: 
`import { FormInput, FormButton } from '../../../components/mainIndex';`

Reply:
Nice suggestion, thanks will implement this